guide.md

```diff
@@ -273,88 +275,88 @@ Finally note that this guide does mention and even recommends various commercial
 -   [Donations:]
 -   [Helping others staying anonymous:]
 -   [Acknowledgments:]
--   [Appendix A: Windows Installation]
-    -   [Installation:][10]
+-   [Appendix A: Windows Installation:]
+    -   [Windows Setup:]
     -   [Privacy Settings:]
--   [Appendix B: Windows Additional Privacy Settings]
--   [Appendix C: Windows Installation Media Creation]
--   [Appendix D: Using System Rescue to securely wipe an SSD drive.]
--   [Appendix E: Clonezilla]
--   [Appendix F: Diskpart]
--   [Appendix G: Safe Browser on the Host OS]
+-   [Appendix B: Windows Additional Privacy Settings:]
+-   [Appendix C: Windows 10 and 11 Installation Media Creation:]
+-   [Appendix D: Using System Rescue to securely wipe an SSD drive:]
+-   [Appendix E: Clonezilla:]
+-   [Appendix F: Diskpart:]
+-   [Appendix G: Safe Browser on the Host OS:]
     -   [If you can use Tor:]
     -   [If you cannot use Tor:]
--   [Appendix H: Windows Cleaning Tools]
+-   [Appendix H: Windows Cleaning Tools:]
 -   [Appendix I: Using ShredOS to securely wipe an HDD drive:]
-    -   [Windows:][11]
-    -   [Linux:]
+    -   [ShredOS on Windows:]
+    -   [ShredOS on Linux:]
 -   [Appendix J: Manufacturer tools for Wiping HDD and SSD drives:]
     -   [Tools that provide a boot disk for wiping from boot:]
-    -   [Tools that provide only support from running OS (for external drives).]
--   [Appendix K: Considerations for using external SSD drives]
-    -   [Windows:][12]
-        -   [Trim Support:]
-        -   [ATA/NVMe Operations (Secure Erase/Sanitize):]
-    -   [Linux:][13]
-        -   [Trim Support:][14]
-        -   [ATA/NVMe Operations (Secure Erase/Sanitize):][15]
-    -   [macOS:][16]
-        -   [Trim Support:][17]
-        -   [ATA/NVMe Operations (Secure Erase/Sanitize):][18]
--   [Appendix L: Creating a mat2-web guest VM for removing metadata from files]
--   [Appendix M: BIOS/UEFI options to wipe disks in various Brands]
--   [Appendix N: Warning about smartphones and smart devices]
--   [Appendix O: Getting an anonymous VPN/Proxy]
+    -   [Tools that provide only support from running OS (for external drives):]
+-   [Appendix K: Considerations for using external SSD drives:]
+    -   [Windows Considerations:]

```

Comment:
"Windows Considerations" seems redundant. Why not just label it "Windows" since it's already a subsection of Appendix K: Considerations for using external SSD drives